composer.json

```diff
@@ -38,9 +38,9 @@
         }
     ],
     "require": {
-        "php": ">= 7.2",
+        "php": "^7.2 || ^8.0",
```

Comment:
Why this change when the Symfony team doesn't recommend this?

Reply:
Do you know why they don't recommend this ?

Reply:
Yeah, that makes sense. I thought `>= 7.2` wouldn't allow for PHP 8, but it obviously does. Will change it back.